packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -588,10 +588,10 @@ class OffsetLayer extends ContainerLayer {
     assert(bounds != null);
     assert(pixelRatio != null);
     final ui.SceneBuilder builder = new ui.SceneBuilder();
-    final Matrix4 transform = new Matrix4.translationValues(bounds.left - offset.dx, bounds.top - offset.dy, 0.0);
-    transform.scale(pixelRatio, pixelRatio);
+    final Matrix4 transform = new Matrix4.identity()
```

Comment:
The previous logic was wrong. The correct value would have been -- bounds.left - offset.dx but that still doesn't handle pixelRatio other than 1 correctly. I don't know if there was a reason that addToScene with the appropriate offset wasn't used but it appears to work fine as well as handling the pixelRatio case correctly. A matrix could be used instead for the offset but the scale operation needs to happen in a different order if that is the case based on experimentation.
Any advice on adding a test for this case? the widget_inspector_test cases cover it but that isn't an ideal place. I can add a golden image test but that would bring in dependencies to the widget layer in test code which is perhaps fine.

Reply:
feel free to write the test at the widgets layer.

Reply:
@tvolkert to review this file.

Reply:
i'd be curious to see whether things get correctly captured if the subtree contains a follower layer, either with the leader layer also in that subtree or with the leader layer in another part of the tree.